framer/Gestures.coffee

```diff
@@ -0,0 +1,47 @@
+# Gesture events
+Gestures = {}
+
+# Pan
+Gestures.Pan = "pan" # This event includes all the other Pan events
+Gestures.PanStart = "panstart"
+Gestures.PanMove = "panmove"
+Gestures.PanEnd = "panend"
+Gestures.PanCancel = "pancancel"
+Gestures.PanLeft = "panleft"
+Gestures.PanRight = "panright"
+Gestures.PanUp = "panup"
+Gestures.PanDown = "pandown"
+
+# Pinch
+Gestures.Pinch = "pinch" # This event includes all the other Pinch events
+Gestures.PinchStart = "pinchstart"
+Gestures.PinchMove = "pinchmove"
+Gestures.PinchEnd = "pinchend"
+Gestures.PinchCancel = "pinchcancel"
+Gestures.PinchIn = "pinchin"
+Gestures.PinchOut = "pinchout"
+
+# Press
+Gestures.Press = "press"
```

Comment:
I just took Hammer naming, but I was thinking that maybe "Press" is not very helpful as the gesture name. Maybe "LongPress" or "PressHold" is more meaningful?